ui/src/app/components/buttons.tsx

```diff
@@ -330,7 +334,7 @@ export const TabButton = ({disabled = false, style = {}, active = false, childre
   >{children}</Clickable>;
 };
 
-export const Link = ({disabled = false, style = {}, children, ...props}) => {
+export const LinkButton = ({disabled = false, style = {}, children, ...props}) => {
```

Comment:
Could you add a comment here about intended usage? Based on the PR it seems like the intended use case for this is: when you want something that looks like a link but isn't one, usually configured with a custom onclick handler.